src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -209,4 +209,10 @@
         "It has been used in the past to supply options to the qstat command.",
         check=lambda line: "QSTAT_OPTIONS" in line,
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="LSF_SERVER as QUEUE_OPTION to the LSF system will be removed in "
+        "the future, and it is not recommended to use this QUEUE_OPTION.",
```

Comment:
I think we should sharpen this text a bit now:

"LSF_SERVER as QUEUE_OPTION is not needed and will be removed in the future. Please remove the configuration line."